src/algorithms/diamondsquare.jl

```diff
@@ -21,48 +23,40 @@ edge midpoints from the nearest two corners and the square's center, where as
 midpoint-displacement only intepolates from the nearest corners (see `MidpointDisplacement`).
 
 """
-struct DiamondSquare <: NeutralLandscapeMaker
-    H::Float64
+@kwdef struct DiamondSquare <: NeutralLandscapeMaker
```

Comment:
I don't usually use @kwdef - does that mean you need to call it as `DiamondSquare(H = 0.2)` to initialize this object? If that's the case I suggest replacing `H` with a more informative variable name like `autocor` or even `autocorrelation` (and the same for all other algs)

Reply:
I think all packages use H, so I would document what it means, and then use the same syntax for constructors as the other methods. This would make the code more consistent.

Reply:
`@kwdef` is basically a shortcut for adding keyword constructors. The single argument methods don't really need them, but they're nice for multi-argument methods, so I standardised that all have both keywords and regular arg constructors to avoid confusion. 

You can still use regular argument constructors - I just forgot to include that in this particular doc, it's in most of the other docs.

And yes the fieldnames as keywords do become more important. I'm not a huge fan of math in struct fields, it tends to make the algorithms impenetrable. `H = alg.autocor` is my usual preference. But if it's common in the package it's probably ok.